spec/features/merchants/items/show_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'merchant items show page' do
+  describe 'as a merchant, when I click on the name of an item from the merchant items index page' do
+    it 'I am taken to that merchants items show page' do
+
+      # visit "/merchants/#{Merchant.first.id}/items"
+
+      # expect(page).to have_link("Item Qui Esse")
+      # click_link("Item Qui Esse")
+      visit "/merchants/#{Merchant.first.id}/items/#{Item.find_by(name: "Item Qui Esse").id}"
```

Comment:
I would recommend changing this to using the rails version of routes. So this I think would be something like `item_show_path(:id)`